configurable_lti_consumer/static/js/configurable_xblock_lti_consumer.js

```diff
@@ -1,9 +1,14 @@
-function configurableLTIConsumerXblockIframeResizerInit() {
-    $(function ($) {
-        LtiConsumerXBlock();  // call inherited lti_consumer javascript initialization
-        $('.ltiLaunchFrame').iFrameResize({ // Initialize IframeResizer on lti_consumer iframe class
-            checkOrigin: false,
-            log: false
-        })
-    });
+function configurableLTIConsumerXblockIframeResizerInit(runtime, element, json_args) {
+    var $element = $("#" + json_args["element_id"]);
+    LtiConsumerXBlock();  // call inherited lti_consumer javascript initialization
+    // Estimate LTI consumer xblock height regarding available width using given ratio
+    // to presize iframe before iframeResizer initialization
+    var ratio = parseFloat($element.find("div").data('ratio'));
+    var $iframe = $element.find('iframe');
+    var width = $('.content-primary,.vert-mod').width();
+    var probable_height = width * ratio;
```

Comment:
You should keep using the notion of `estimation`:
```suggestion
    var estimated_height = width * ratio;
```

Reply:
ok